apps/storefront-e2e/src/support/page-fragments/breadcrumbs.fragment.ts

```diff
@@ -0,0 +1,15 @@
+export class BreadcrumbsFragment {
+  get = () => cy.get('oryx-site-breadcrumbs');
+  lastBreadcrumb = () => this.get().find('a:last-of-type');
+  nthBreadcrumb = (n: number) => this.get().find(`a:nth-of-type(${n})`);
```

Comment:
can we use nthBreadcrumb(-1) for last element?
`this.get().find('a').eq(number) `